src/components/Notify.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+import React from 'react';
+import { Toast } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const StyledMessage: any = styled.span`
+    color: ${(props: any) => (props.type === 'info' ? 'darkgreen' : 'red')};
+`;
+
+interface INotifyProps {
```

Comment:
If you don't export `Props` interface you can name this as `Props`. Because you will not get any names conflicts in others files in future